### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by TristenUrban**
HomePage

Implemented Home Page and sidebar file cleanup

### CODE REVIEW

File: components/basecomponents/SideBarStyles.js
Comment (cmjosephs) [inline components/basecomponents/SideBarStyles.js]: Lets make sure naming stays descriptive. I think I know what AppBar is but it might not be clear to an engineer looking at this for the first time. The whole thing is the app and bar doesn't say much. Maybe add in what the purpose the bar serves like 'headerBar' or something similar.

File: components/basecomponents/SideBarStyles.js
Comment (cmjosephs) [inline components/basecomponents/SideBarStyles.js]: Same as comment above. The app could have multiple drawers so something like 'sideNavDrawer' (or something else) might be more descriptive.

File: components/NavLayout/SideBar.js
Comment (cmjosephs) [inline components/NavLayout/SideBar.js:22]: Great job separating out the styled base components 😃 . Now when navigate files, we see our component that gets rendered  immediately after our imports which helps with code readability and consequently future debugging.

File: components/NavLayout/SideBar.js
Comment (cmjosephs) [inline components/NavLayout/SideBar.js:42]: Up to your discretion, but we can further modularize this. Could add a line of code above 
`const iconColorStyle = { color: '#FFFFFF' }` 
and for each icon component set
`<SomeIcon style={iconColorStyle} />`
This keeps our code a little more DRY.

File: components/NavLayout/SideBar.js
Comment (cmjosephs) [inline components/NavLayout/SideBar.js:109]: Could we wrap the <Link /> around the text and icon? If you can't for some reason, drop a comment in the code explaining why.

File: components/basecomponents/MainCard.js
Comment (cmjosephs) [inline components/basecomponents/MainCard.js:1]: If leaving an empty-ish component like this, have a more descriptive component name and drop a comment explaining what its for in the mean time. I'm not sure what MainCard by the name is and theres no code telling me what its supposed to be for.

File: components/basecomponents/SideBarStyles.js
Comment (cmjosephs) [inline components/basecomponents/SideBarStyles.js:70]: Again, great work separating the files out.

File: utils/styles/globals.css
Comment (cmjosephs) [inline utils/styles/globals.css:31]: Double check that this margin is dynamic and works properly with that bar on the top. I'm sure it works but px are a hard unit of measurement so we have to be extra sure its stays responsive with changing page sizes and mobile screens.

**Review by cmjosephs — APPROVED** [review]
Great work, code's looking much better. Going to approve but please read through additional comments. Please also get one other approval before merging.

**Comment by cmjosephs** [comment]
Also more description in the PRs. Screenshots/vids are great but I'd like to see more description of what was done in the code at a high level.

File: components/NavLayout/SideBar.js
Comment (TristenUrban) [reply components/NavLayout/SideBar.js:109]: Yeah! we tried that in the beginning, but the code breaks when everything is included in one line, so the separation is intentional

**Review by cmjosephs — APPROVED** [review]
Great work!

**Pull request merged**
